### PR TITLE
function isPasswordAutoCompleteDisabled() commented

### PR DESCRIPTION
**Pull request by feelcoder**
Resolve issue #3440. I decided to first comment the function. After I deleted in some places where it was being called and also commented in some files where it was being called.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: e036a9b9851fbaa91883e6bbbbeda965dbbbcf28
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

File: app/scripts/templates/complete_reset_password.mustache [hunk 1]
```diff
@@ -46,7 +46,7 @@
 
       <div class="input-row password-row">
         <label class="label-helper"></label>
-        <input type="password" class="password" id="password" placeholder="{{#t}}New password{{/t}}" pattern=".{8,}" autofocus required {{#isPasswordAutoCompleteDisabled}}autocomplete="off"{{/isPasswordAutoCompleteDisabled}} />
+        <input type="password" class="password" id="password" placeholder="{{#t}}New password{{/t}}" pattern=".{8,}" autofocus
```
Comment (pdehaan) [inline app/scripts/templates/complete_reset_password.mustache]: Missing closing `/>`

File: app/scripts/views/mixins/password-mixin.js [hunk 2]
```diff
@@ -39,17 +42,12 @@ define(function (require, exports, module) {
...
-          passwordField.attr('type', 'text').attr('autocomplete', 'off')
-            .attr('autocorrect', 'off').attr('autocapitalize', 'off');
+          passwordField.attr('type', 'text').attr('autocorrect', 'off').attr('autocapitalize', 'off');
           this.logViewEvent('password.visible');
         } else {
           passwordField.attr('type', 'password');
-          if (this.isPasswordAutoCompleteDisabled()) {
-            passwordField.attr('autocomplete', 'off');
-          } else {
             passwordField.removeAttr('autocomplete')
```
Comment (pdehaan) [inline app/scripts/views/mixins/password-mixin.js]: Indentation looks a bit off on line 49.

**Comment by vladikoff** [comment]
@feelcoder Hey! looks like @pdehaan gave you a quick review, you can keep working in this pull request by just pushing new commits to it. You don't need to close it. See this guide here: https://www.atlassian.com/git/tutorials/making-a-pull-request/example and more here https://www.atlassian.com/git/tutorials/syncing/git-push

The linked guide describes the "follow-up commit" scenario. You basically make more changes to this branch, `git commit` and push to it so we can see the updates.

**Comment by feelcoder** [comment]
@vladikoff, I noticed I did not use correct naming format, so I had to review it and I did a new pull request but it is not passing all the test, precisely the npm test.

**Pull request closed without merging**